src/menuConfig.js

```diff
@@ -533,6 +533,7 @@ export const InterPro /*: Array<Object> */ = [
 ];
 
 const _NOT_MEMBER_DBS = [
```

Comment:
Could this be stored a boolean in the MySQL database and accessed through the API?